core/tools/dnsmasq/bootstrap.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+# Exit if something goes wrong
+set -e
+
+# Dnsmasq provides DHCP and DNS servers for small networks
+echo "Installing dnsmasq."
+apt install -y --no-install-recommends dnsmasq
+
+# Stops dnsmasq service. It will be run independently.
```

Comment:
this looks weird. is the install step starting the service? even then, this runs in build time, I expected something analog to `service disable", not stop.

can you make explain better in comments?

Reply:
I'm not much familiar with non-systemd managers, as this one, and it doesn't have 'disable'.

During tests I remember to find out that maybe the stop command also disabled the service, but I can be wrong. Will test without it.

Reply:
Testes again and its not needed indeed. Removed.